src/rest/src/main/java/org/geoserver/rest/util/IOUtils.java

```diff
@@ -769,7 +769,7 @@ public static void inflate(ZipFile archive, Resource outputDirectory, String fil
      */
     public static void inflate(ZipFile archive, Resource outputDirectory, String fileName,
             String workspace, String store, List<Resource> files,
-            boolean external) throws IOException {
+            boolean external, boolean saveFile) throws IOException {
```

Comment:
Missing javadoc for the new parameter

Reply:
Also, this is public API, there should be a method without the new parameter and a default with the same behavior as before (for backwards compatibility)

Reply:
@aaime ok thanks, will fix ASAP.